client/src/components/ProductDetails.jsx

```diff
@@ -41,10 +41,14 @@ class ProductDetails extends React.Component {
         <FeatureHeader>Features</FeatureHeader>
         <DetailsContainer>
           <DetailsRow>
-            <div>Feature Title</div>
-            <div>Feature Description</div>
-            <div>Feature Title</div>
-            <div>Feature Description</div>
+            <div>
+              <div>{this.props.details.feature1_title}</div>
```

Comment:
@masoncpott you could probably de-structure your this.props.details up top and use the variables here. Just my two cents.